src/Symfony/Component/Translation/Translator.php

```diff
@@ -494,13 +463,14 @@ private function getFallbackContent(MessageCatalogue $catalogue)
         return $fallbackContent;
     }
 
-    private function getResourcesHash($locale)
+    private function getCatalogueCachePath($locale)
     {
-        if (!isset($this->resources[$locale])) {
-            return '';
-        }
+        $catalogueHash = sha1(serialize(array(
+            'resources' => isset($this->resources[$locale]) ? $this->resources[$locale] : array(),
```

Comment:
@aitboudad While backporting the Translator test case refactorings to the 2.6 branch, I noticed that the test for this fails because we don't have that fix on 2.6 yet (different resources still use same cache).

Is that something you would want to fix on 2.6 as well?